dashboard/config/locales/en.yml

```diff
@@ -1424,6 +1424,9 @@ en:
       form_instructions_2: "Full guides for obtaining your LMS Client ID, using LMS/LTI integrations, and our list of supported Learning Management Systems <a href='https://code.org/contact'>can be found here.</a>"
       form_instructions_3: "Not seeing your LMS in our list, or running into issues with your installation? Check out our <a href=''>guides</a>, or <a href='https://code.org/contact'>contact us</a> for additional help"
       guide_instructions: "Full guides for installing and our LTI integration <a href=''>can be found here.</a> Running into issues? Feel free to <a href=''>contact us</a> for additional support."
+      mailer_confirmation_subject: "Code.org has completed your LMS registration request"
+      mailer_confirmation_body: "Great news! We’ve registered your LMS with Code.org. You should now be able to use our Code.org LMS integrations. Please feel free to review & share our <a href=''>LMS integration guides</a>, or feel free to browse our <a href=''>course offerings</a>."
+      mailer_confirmation_signature: "Thank you and welcome again!"
```

Comment:
What does "welcome again" mean here? Is it referring to them being welcomed once upon initial account registration, then again upon registering their LMS?

Reply:
@alivira ^^

Reply:
This was probably due to this messaging being the combination of 3 originally planned emails. Since we never actually say "welcome" for the first time, I'd probably drop the "again" here and just say: "Thank you, and welcome!"